### PR TITLE
docker-rm: add page; docker-container-{remove, rm}: add alias pages

### PR DESCRIPTION
**Pull request by aniketpuro**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/common/docker-rm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# docker rm
+
+> Remove one or more containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/rm/>.
```
Comment (gutjuri) [inline pages/common/docker-rm.md:4]:
```suggestion
> More information: <https://docs.docker.com/engine/reference/commandline/rm>.
```

File: pages/common/docker-rm.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Remove containers:
+
+`docker rm {{container1 container2 ...}}`
+
+- Force remove a container:
+
+`docker rm --force {{container1 container2 ...}}`
+
+- Remove container and it's volume:
```
Comment (gutjuri) [inline pages/common/docker-rm.md:18]:
```suggestion
- Remove container and its volumes:
```

**Review by gutjuri — COMMENTED** [review]
Thanks, I have two suggestions

File: pages/common/docker-rm.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`docker rm {{container1 container2 ...}}`
+
+- Force remove a container:
+
+`docker rm --force {{container1 container2 ...}}`
+
+- Remove container and its volumes:
+
+`docker rm --volumes {{container}}`
```
Comment (kbdharun) [inline pages/common/docker-rm.md:20]:
```suggestion
- Remove containers:

`docker rm {{container1 container2 ...}}`

- Force remove a container:

`docker rm --force {{container1 container2 ...}}`

- Remove a container and its volumes:

`docker rm --volumes {{container}}`

- Display help:

`docker rm`
```

Similar to https://github.com/tldr-pages/tldr/pull/11127/files#r1367765109

Ref.

```zsh
➜  ~ sudo docker rm
Place your right index finger on the fingerprint reader
"docker rm" requires at least 1 argument.
See 'docker rm --help'.

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers
➜  ~ sudo docker rm --help

Usage:  docker rm [OPTIONS] CONTAINER [CONTAINER...]

Remove one or more containers

Aliases:
  docker container rm, docker container remove, docker rm

Options:
  -f, --force     Force the removal of a running
                  container (uses SIGKILL)
  -l, --link      Remove the specified link
  -v, --volumes   Remove anonymous volumes associated
                  with the container
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**